client/src/components/InitialPromptCreationScreen/InitialPromptCreationScreen.jsx

```diff
@@ -40,12 +76,29 @@ class InitialPromptCreationScreen extends React.Component {
   }
 
   render() {
-    const { error } = this.state;
+    const {
+      noun, adjective1, adjective2, error,
+    } = this.state;
     return (
       <div className="InitialPromptCreationScreen">
-        Adjective, adjective, noun.
+        <h3>Enter the prompts for other players to draw</h3>
+        <label htmlFor="noun">
+          Noun
```

Comment:
Minor: Would it make sense to put the Noun at the end since you usually say `lusty orange narwhal` vs `narwhal lusty orange`?

Reply:
Yupyup